pytest/tests/sanity/lightclnt.py

```diff
@@ -96,7 +97,7 @@ def get_up_to(from_, to):
 
     for i in range(from_, to + 1):
         hash_ = height_to_hash[i]
-        print(i, hash_, hash_to_epoch[hash_], hash_to_next_epoch[hash_])
+        logger.info(i, hash_, hash_to_epoch[hash_], hash_to_next_epoch[hash_])
```

Comment:
This needs to be changed to use format string. Here and below in this file.

Reply:
Wow, I thought I got all these with my regex matching, but guess not

Reply:
https://xkcd.com/1171/